nifi-nar-bundles/nifi-graph-bundle/nifi-other-graph-services/src/main/java/org/apache/nifi/graph/TinkerpopClientService.java

```diff
@@ -140,7 +139,7 @@ public class TinkerpopClientService extends AbstractControllerService implements
             .required(true)
             .defaultValue("/gremlin")
             .addValidator(StandardValidators.NON_EMPTY_EL_VALIDATOR)
```

Comment:
Is it possible to configure a Gremlin server such that the path is empty? If so we should replace this validator with either Validators.TRUE or something like that

Reply:
@mattyb149  -- https://tinkerpop.apache.org/javadocs/current/core/org/apache/tinkerpop/gremlin/driver/Cluster.Builder.html#path(java.lang.String) looks like it defaults to "/gremlin" if not provided, so I think we can safely change this validator.